src/Command/MarkMigrated.php

```diff
@@ -53,10 +53,22 @@ protected function execute(InputInterface $input, OutputInterface $output)
     {
         $this->setInput($input);
         $this->bootstrap($input, $output);
+        $Adapter = $this->getManager()->getEnvironment('default')->getAdapter();
```

Comment:
@lorenzo I thought variables referencing an object had to start with a capital letter (as per Cake documented coding standards). 
(I don't mind changing it at all, I'm just asking for future contributions)